### ISSUE TITLE
Allow gating signups on user con profile fields

### ISSUE THREAD

**Issue by nbudin** [neinteractiveliterature/intercode#8627]
For example, Be-Con would like to make it so that people's signup choices won't be processed until their vax check has been verified.

**Comment by vogon999** [comment]
I have a similar request, I was going to request that the site allow sales of badges to be gated by whether the attendee is verified (this would need to be an adjustable setting, of course, the idea would be that Intercon flips over to requiring vax verification prior to badge sale on Jan 1st).  

I don't think it's an urgent request and is more of a nice to have, but I thought I'd add it here in case there are changes being made to the field and it's possible to do both with the edit.  Thanks!
